commands/init.lua

```diff
@@ -0,0 +1,100 @@
+local core = require("core")()
+local prompt = require("prompt")(require("pretty-print"))
+local fs = require("coro-fs")
+local env = require("env")
+local log = require("log").log
+local pathJoin = require("luvi").path.join
+local cwd = require('uv').cwd()
+
+local config = core.config
+
+local function getOutput()
+  local output = prompt("Output to package.lua (1), or a init.lua (2)?")
+  -- response was blank, run again
+  if not output then
+    return getOutput()
+  end
+  -- fail on any other options
+  if not output:match('[1-2]') then
+    log("Error", "You must select a valid option. [1 or 2]")
+    return getOutput()
+  else
+    if output == "1" then
+      output = "package.lua"
+      log("Creating", "package.lua")
+    elseif output == "2" then
+      output = "init.lua"
+      log("Creating", "init.lua")
+    end
+  end
+  return output
+end
+
+local output = getOutput()
+
+local home = env.get("HOME") or env.get("HOMEPATH") or ""
+local ini
+local function getConfig(name)
+  ini = ini or fs.readFile(pathJoin(home, ".gitconfig"))
+  if not ini then return end
+  local section
+  for line in ini:gmatch("[^\n]+") do
+    local s = line:match("^%[([^%]]+)%]$")
+    if s then
+      section = s
+    else
+      local key, value = line:match("^%s*(%w+)%s*=%s*(.+)$")
+      if key and section .. "." .. key == name then
+        if tonumber(value) then return tonumber(value) end
+        if value == "true" then return true end
+        if value == "false" then return false end
+        return value
+      end
+    end
+  end
+end
+
+local projectName = prompt("Project Name", config["username"] .. "/project-name")
+local projectVersion = prompt("Version", "0.0.1")
+local projectDescription = prompt("Description", "A simple description of my little package.")
+local projectTags = prompt("Tags (Comma Separated)", "lua, lit, luvit")
+local authorName = prompt("Author Name", getConfig("user.name"))
+local authorEmail = prompt("Author Email", getConfig("user.email"))
+local projectLicense = prompt("License", "MIT")
+local projectHomepage = prompt("Homepage", "https://github.com/" .. projectName)
+
+-- trim and wrap words in quotes
+function makeTags(csv)
+  local tags = "{ "
+  for word in csv:gmatch('([^,]+)') do
+    tags = tags .. "\"" .. word:gsub("^%s*(.-)%s*$", "%1") .. "\", "
+  end
+  -- trim trailing comma and space
+  tags = tags:sub(0, (#tags - 2))
+  return tags .. " }"
+end
+
+local data = ""
+
+-- manually craft strings to control formatting
+if output == "init.lua" then
+  data = "exports.name = \"" .. projectName .. "\"\nexports.version = \"" .. projectVersion .. "\"\nexports.dependencies = {}\nexports.description = \"" .. projectDescription .. "\"\nexports.tags = " .. makeTags(projectTags) .. "\nexports.license = \"" .. projectLicense .. "\"\nexports.author = {\n  name = \"" .. authorName .. "\",\n  email = \"" .. authorEmail .. "\"\n}\nexports.homepage = \"" .. projectHomepage .. "\"\n"
```

Comment:
Hi James! Thank you for working on this. This could be refactored to be something like:
```lua
local t = {
  name = projectName,
  version = projectVersion,
  dependencies = {},
  tags = makeTags(projectTags),
  etc... }

for k, v in pairs(t) do
  if type(v) == "string" then
    print(string.format("exports.%s = \"%s", k, v))
  if type(v) == "table" then
    print(string.format("exports.%s = {}, k))
  end
end
```

Alternatively, we could use a multi line template:
```
tmpl = [[
exports.name = "%s"
exports.version = "%s"
exports.dependencies = {}
]]
```
then pass it to string.format()

This feature will be super useful. personally, I like the template format.